boards/mega-xplained/Makefile.features

```diff
@@ -2,6 +2,8 @@
 FEATURES_PROVIDED += periph_adc
 FEATURES_PROVIDED += periph_gpio
 FEATURES_PROVIDED += periph_i2c
+FEATURES_PROVIDED += periph_rtc
+FEATURES_PROVIDED += periph_rtt
```

Comment:
For other atmega to work, you need to add these into it's Makefile.features.

Reply:
Added support to jiminy-mega256rfr2. See comment below review.